replicate/common/generic-replicator.go

```diff
@@ -356,7 +356,7 @@ func (r *GenericReplicator) replicateResourceToNamespaces(obj interface{}, targe
 	cacheKey := MustGetKey(obj)
 
 	for _, namespace := range targets {
-		if err := r.UpdateFuncs.ReplicateObjectTo(obj, &namespace); err != nil {
+		if err = r.UpdateFuncs.ReplicateObjectTo(obj, &namespace); err != nil {
```

Comment:
Uuh... Is this why no error was reported when the replication failed? 🤦

Reply:
Yeah, the function just returned nil regardless